ext/MakieTeXCairoMakieExt.jl

```diff
@@ -23,9 +23,9 @@ CairoMakie.cairo_scatter_marker(v::NTuple{N, <: MakieTeX.AbstractDocument}) wher
 
 # # Teximg
 
-# Override `is_cairomakie_atomic_plot` to allow `TeXImg` to remain a unit,
+# Override `is_cairomakie_atomic_plot` to allow `TeXImg` and `TypstImg` to remain a unit,
 # instead of auto-decomposing into its component scatter plot.
-CairoMakie.is_cairomakie_atomic_plot(plot::TeXImg) = true
+CairoMakie.is_cairomakie_atomic_plot(plot::Union{TeXImg, TypstImg}) = true
```

Comment:
In this case I think we should let TeXImg be the main/only plot type, similarly for LTeX - they would just accept TypstDocuments as well.

Reply:
[Got it!](https://github.com/JuliaPlots/MakieTeX.jl/pull/48/commits/57245bbff61659995a8bb4ff1252915e97a04354#diff-22b3e476a39ae93a6a686b3a90fcbda7cec1f10ca853e56f3db0c2ced2176ed7L26-L28)

Reply:
Just to clarify, I meant that we should remove the TypstImg and LTypst entirely, since they don't have any differentiating factors from LTeX and the two would get a bit confusing IMO.  If there is some additional value to be gained by having separate recipes then we can look into this again, but for now it seems easier to just have one canonical recipe to plot things with...

Reply:
Oh, thank you for elaborating. I didn't actually know what it did and tried to revert it.